src/index.js

```diff
@@ -120,6 +121,12 @@ function App() {
           component={() => <div>"My Mentees" Component goes here</div>}
         />
 
+        <PrivateRoute
+          path="/resources"
+          redirect="/dashboard"
+          allowRoles={[1, 2, 3, 4]}
+          component={ManageResources}
+        />
```

Comment:
Please add a line break here between these lines so that it matches the rest of the file's composition.

Reply:
Added line break to match formatting